### PR TITLE
Add new field to Career Dev and IEP

### PR DESCRIPTION
**Pull request by magonzalez9**

### CODE REVIEW

File: src/components/EditCareerDevForm.js [hunk 1]
```diff
@@ -25,20 +27,23 @@ class SkillsCareerDevForm extends Component {
 
   handleBoolChange = key => value => this.setKey(key, value);
   handleStringChange = key => event => this.setKey(key, event.target.value);
-
+  handleSchoolGoals = key => event => this.setKey(key, event.target.value);
```
Comment (ryan-peterson-emberex) [inline src/components/EditCareerDevForm.js]: Isn't needed.

File: src/components/EditCareerDevForm.js [hunk 2]
```diff
@@ -25,20 +27,23 @@ class SkillsCareerDevForm extends Component {
...
-
+  handleSchoolGoals = key => event => this.setKey(key, event.target.value); 
   handleAttendedIepMeetingChange = attended => {
     this.setKey('attendedIepMeeting', attended);
     if(!attended) this.setKey('iepRole', null);
   };
+
   handleIepRoleChange = this.handleStringChange('iepRole')
   handleHasGraduationPlanChange = this.handleBoolChange('hasGraduationPlan');
+  handlePostSchoolGoals = this.handleSchoolGoals('postSchoolGoals');
```
Comment (ryan-peterson-emberex) [inline src/components/EditCareerDevForm.js]: Should be `handlePostSchoolGoals = this.handleStringChange('postSchoolGoals');`

The handleStringChange returns a closure that will change one of the variables on the EditCareerDevForm with the same name as the `key` string. Admittedly, I've never seen this style before.

File: src/components/EditCareerDevForm.js [hunk 3]
```diff
@@ -98,6 +105,13 @@ class SkillsCareerDevForm extends Component {
           </FormColumn>
           <FormColumn/>
         </FormRow>
+        <FormRow>
+          <FormColumn>
+            <Label>Post-school Goals</Label>
```
Comment (ryan-peterson-emberex) [inline src/components/EditCareerDevForm.js]: For consistency, Goals should be lowercase.

File: src/components/StudentRisksView.js [hunk 4]
```diff
@@ -93,6 +93,7 @@ class StudentRisksView extends Component {
             <ValueRow name="IEP Meeting Role" value={student.iepRole}/>
           }
           <ValueRow name="Career development/Graduation plan" value={student.hasGraduationPlan}/>
+          <ValueRow name="Post-school Goals" value={student.postSchoolGoals}/>
```
Comment (ryan-peterson-emberex) [inline src/components/StudentRisksView.js]: For consistency, Goals should be lowercase.

File: src/components/EditCareerDevForm.js [hunk 5]
```diff
@@ -27,15 +27,14 @@ class SkillsCareerDevForm extends Component {
...
-  handleSchoolGoals = key => event => this.setKey(key, event.target.value); 
   handleAttendedIepMeetingChange = attended => {
     this.setKey('attendedIepMeeting', attended);
     if(!attended) this.setKey('iepRole', null);
   };
 
   handleIepRoleChange = this.handleStringChange('iepRole')
   handleHasGraduationPlanChange = this.handleBoolChange('hasGraduationPlan');
-  handlePostSchoolGoals = this.handleSchoolGoals('postSchoolGoals'); 
+  handlePostSchoolGoals = this.handleStringChange('postSchoolGoals');
```
Comment (ryan-peterson-emberex) [inline src/components/EditCareerDevForm.js]: :+1:

**Comment by jf-emberex** [comment]
No need to wait for Jimmy's review since he is out.

**Pull request merged**